train_and_eval/train/train_imagenet.py

```diff
@@ -0,0 +1,607 @@
+import torch as ch
```

Comment:
We should write some header comment saying this code was taken directly from FFCV repo? @adityakusupati

Reply:
Added.